# Exceptions-String_to_Integer

Another problem with exception.

First I tried to solve this problem with 'atoi', but 'atoi' throws 0 as exception.
Therefore I used stoi function.
